op-node/rollup/derive/engine_queue.go

```diff
@@ -682,7 +682,8 @@ func (eq *EngineQueue) Reset(ctx context.Context, _ eth.L1BlockRef, _ eth.System
 	return io.EOF
 }
 
-// GetUnsafeQueueGap retrieves the current [start, end] range of the gap between the tip of the unsafe priority queue and the unsafe head.
+// GetUnsafeQueueGap retrieves the current [start, end) range (incl. start, excl. end)
```

Comment:
the `incl. start, excl. end` description here is messing with my head. My understanding is that we're returning the start and end of the range of blocks we're missing and we'd need to request all of those blocks, including the start and end.

Reply:
including the start, excluding the end. We could also redefine it as `start, count` if that helps, that's what L1 uses: https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#beaconblocksbyrange

Reply:
oh right, yes. Hence the need for the + 1. I just tied myself in knots for some reason...